### DISCUSSION TITLE
localhost redirect_url is not working in brave browser

### DISCUSSION THREAD

**Discussion by RakeshSangem** [supabase/supabase#20440] [Contribute to Supabase]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

when using google auth `redirectTo` property it's not redirecting to the page and giving the below error. I also added the redirect url in the supabase auth dashboard. 

When check it on the `Google Chrome` it's working fine.

## Screenshots

<img width="686" alt="image" src="https://github.com/supabase/supabase/assets/107752425/6e078dc2-71b7-40da-9041-7875d7832c38">

## System information

- OS: macOS
- Brave browser

**Comment by gregnr** [comment]
Hey @RakeshSangem! Can you confirm that http://localhost:3000/user/profile is accessible on its own and renders correctly in your dev environment? Hard to say for sure without trying it, but based on your screenshot it seems that the redirect is happening but the page just isn't available/being served.

Also I noticed you mentioned `redirecTo` (misspelling of `redirectTo`) - I assume this was just a typo in this issue, but worth double checking your code as well.

**Comment by RakeshSangem** [comment]
Hello @gregnr , I can assure you that http://localhost:3000/user/profile is accessible, and I previously encountered the same problem before I added redirect URL in URL configurations and it worked. I don't why we are facing the same issue again, and it is not giving any issue in the production environment.

**Comment by Zavierrr** [comment]
> Hey @RakeshSangem! Can you confirm that http://localhost:3000/user/profile is accessible on its own and renders correctly in your dev environment? Hard to say for sure without trying it, but based on your screenshot it seems that the redirect is happening but the page just isn't available/being served.
> 
> Also I noticed you mentioned `redirecTo` (misspelling of `redirectTo`) - I assume this was just a typo in this issue, but worth double checking your code as well.

I also encountered a similar problem. It seems that the redirection did not work.
![image](https://github.com/supabase/supabase/assets/48075834/7ea83f3d-250c-48fd-8e65-2a2f5321371e)
